src/flow/index.js

```diff
@@ -0,0 +1,268 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { FlowChartWithState } from '@mrblenny/react-flow-chart'
+import dagre from 'dagre'
+
+import Container from './container'
+import Port from './port'
+import NodeInner from './node-inner'
+
+
+export const stepProps = Object.freeze({
```

Comment:
could make stepProps a constant that you import here and in dagre.js

Reply:
this and a bunch of other repeated codes were intentional as these are highly experimental sub-components (basically trialing like 2-3 underlying libraries a week).

copy/paste + subtle modifications were intentionally done to avoid over-abstraction (usually through such as the suggestion of passing `report` as a property, and rewire with flow control such as `if/else`).

going to consolidate and eventually when we settle on a solid visualization solution